handbook/bizops/index.md

```diff
@@ -89,40 +72,11 @@ Every underlying data source (not chart!) is assumed to always be up-to-date unl
 
 #### HubSpot
 
-Every Sunday night or Monday morning, [BizOps scripts](https://github.com/sourcegraph/analytics/tree/master/HubSpot%20ETL) are triggered to retrieve data from HubSpot and upload it to BigQuery. As a result, HubSpot data in Looker is updated weekly, not daily.  If you need the latest data during the week, mention @ericbm in the #analytics channel in Slack.
+[The HubSpot data pipeline](https://github.com/sourcegraph/analytics/tree/master/HubSpot%20ETL) is updated once per day (in the afternoon PST).  If you need the latest data at any time, post in the #analytics channel in Slack and the BizOps team can run the pipeline manually. 
 
 [Lead-generation](https://docs.google.com/spreadsheets/d/16S3xlcY7DmpcfKZYD-3VHUsaPLiYHyisu8cD_gZpv0Q/edit#gid=0) from HubSpot is triggered by Zapier and is always up-to-date. [Marketing operations](../marketing/marketing_operations.md#maintaining-data-pipelines) has more information on lead-gen events. 
 
 ## Using Looker
 
-[Looker](https://sourcegraph.looker.com/) is a self-service tool with many pre-built reports and visualizations. Read our instructions for [getting started with Looker](#using-looker) and reach out in the #analytics Slack channel if you have any questions.
-
-### Sourcegraph quick links
```

Comment:
I still think some of these would be helpful to have here. The onboarding links are all videos, so it can take a while to figure out what the useful dashboards are.